SU2_CFD/src/variables/CIncEulerVariable.cpp

```diff
@@ -76,7 +78,7 @@ bool CIncEulerVariable::SetPrimVar(unsigned long iPoint, CFluidModel *FluidModel
   /*--- Set the value of the temperature directly ---*/
 
   su2double Temperature = Solution(iPoint, nDim+1);
-  const auto check_temp = SetTemperature(iPoint, Temperature);
+  const auto check_temp = SetTemperature(iPoint, Temperature, TemperatureLimits);
```

Comment:
Do we get the warnings about the number of unphysical points when the bounds are active?

Reply:
Thanks! I will try a case with tighter limits to check it.